tests/lib/test_lib.py

```diff
@@ -45,7 +45,7 @@ def test_correct_pip_version(script):
     # distribution
     pip_folder_outputed = re.match(
         r"pip \d+(\.[\d]+)+(\.?(b|rc|dev|pre|post)\d+)? from (.*) "
-        r"\(python \d(.[\d])+\)$",
+        r"\(python \d(.[\d]+)+\)$",
```

Comment:
Should probably allow multi-digits for the major version too (though python 10.x is not particularly imminent)

```suggestion
        r"\(python \d+(\.[\d]+)+\)$",
```